.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: node_js
 node_js:
 - 5.6
-before_script:
-- gem install percy-cli
+addons:
+  apt:
+    packages:
+    - oracle-java8-installer
+before_install:
+- export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+- export PATH=$JAVA_HOME/bin:$PATH
 after_success:
-- '[ "$TRAVIS_SECURE_ENV_VARS" = "true" ] && npm run perceptual-diff'
 sudo: false
 deploy:
   provider: npm
```

Comment:
You can drop the NPM deploy -- all the WC stuff uses Bower exclusively. Otherwise things get really messed up with the unreliable location of the dependencies.